taskt/Core/Automation/Commands/File Operation Commands/FileExtractionCommand.cs

```diff
@@ -2,109 +2,113 @@
+        [SampleUsage(@"C:\temp\myfile.zip || https://temp.com/myfile.zip || {vFileSourcePath}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowFileSelectionHelper)]
+        public string v_FilePathOrigin { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the file path to be send after extract")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFileSelectionHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter or Select the path to the applicable file or enter file URL.")]
-        [Attributes.PropertyAttributes.SampleUsage(@"C:\temp\ or [vFilePath]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Extracted File(s) Directory Path")]
+        [InputSpecification("Enter or Select the Folder Path / URL to move extracted file(s) to.")]
+        [SampleUsage(@"C:\temp\ || {vFilesPath}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)]
         public string v_PathDestination { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please select the variable to receive the list of extract files")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable from the variable list")]
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required to pre-define your variables, however, it is highly recommended.")]
-        public string v_applyToVariableName { get; set; }
+        [PropertyDescription("Output Extracted File(s) Variable")]
+        [InputSpecification("Select or provide a variable from the variable list.")]
+        [SampleUsage("vUserVariable")]
+        [Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required" +
+                 " to pre-define your variables; however, it is highly recommended.")]
+        public string v_OutputUserVariableName { get; set; }
 
         public FileExtractionCommand()
         {
-            this.CommandName = "FileExtractionCommand";
-            this.SelectionName = "File Extraction";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "FileExtractionCommand";
```

Comment:
Should we rename this command to Extract Files?

Reply:
renamed